__tests__/app.test.js

```diff
@@ -305,4 +307,119 @@ describe('/api/articles', () => {
         });
 
     })
-})
\ No newline at end of file
+})
+
+describe('/api/articles/:article_id/comments', () => {
+    const newComment = {
+        username: 'butter_bridge',
+        body: 'my comment'
+    }
+    describe('HAPPY PATH POST /api/articles/:article_id/comments', () => {
+        test('201 CREATED - Should add comment inputted with correct properties/values & return it', () => {
+
+            return request(app)
+                .post('/api/articles/1/comments')
+                .send(newComment)
+                .expect(201)
+                .then((response) => {
+                    expect(response.body.comment).toEqual({
```

Comment:
Would still be using `expect.objectContaining` here too to account for additional properties in the future